students/uss_tomasz/lesson_09_reading_and_writing_files/multiclipboard/msb.pyw

```diff
@@ -0,0 +1,45 @@
+# msb.pyw - saves and loads pieces of text to the clipboard.
+# usage: py.exe msb.pyw save <keyword> - saves clipboard to keyword.
+#        py.exe mcb.pyw <keyword> - loads keyword to clipboard.
+#        py.exe mcb.pyw list - loads all keywords to clipboard.
+#        py.exe mcb.pyw delete <keyword> - delete keyword from shelve
+#        py.exe mcb.pyw delete - delete all content from shelve
+
+import pyperclip
+import shelve
+import sys
+mcb_shelf = shelve.open('mcb')
+if len(sys.argv) < 2:
```

Comment:
It's a popular pattern to check execution preconditions upfront and exit if not fulfilled. You could do it this way:
```
if len(sys.argv) < 2:
    print('Missing command')
    sys.exit(1)

mcb_shelf = shelve.open('mcb')
command = sys.argv[1].lower()
...
```
This way you save 1 indentation level for like ~30 lines